releng/org.eclipse.n4js.parent/pom.xml

```diff
@@ -247,8 +247,11 @@ Contributors:
 							is not considered by Tycho/maven together with our harsh settings for optional warnings/errors;
 							therefore: do not use this at the moment!)
 							Note: by default, the setting is enabled.
-						<useProjectSettings>true</useProjectSettings>
 						-->
+						<useProjectSettings>true</useProjectSettings>
```

Comment:
add rationale for using this (beyond just "it's more consistent") as comment here